draft-steele-cose-merkle-tree-proofs.md

```diff
@@ -341,6 +332,17 @@ See the privacy considerations section of:
 - {{-certificate-transparency-v2}}
 - {{-COSE}}
 
+Although the word transparency implies to some degree read access,
```

Comment:
I attempt to add some text about privacy here, comments welcome

Reply:
:+1: